gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -428,40 +410,22 @@ public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOpti
             .build();
 
     AbstractGoogleAsyncWriteChannel<?> channel;
-    if (storageOptions.isGrpcEnabled()) {
-      String requesterPaysProjectId = null;
-      if (requesterShouldPay(resourceId.getBucketName())) {
-        requesterPaysProjectId = storageOptions.getRequesterPaysOptions().getProjectId();
-      }
-      channel =
-          new GoogleCloudStorageGrpcWriteChannel(
-              storageStubProvider,
-              backgroundTasksThreadPool,
-              storageOptions,
-              resourceId,
-              options,
-              watchdog,
-              writeConditions,
-              requesterPaysProjectId,
-              BackOffFactory.DEFAULT);
-    } else {
-      channel =
-          new GoogleCloudStorageWriteChannel(
-              storage,
-              clientRequestHelper,
-              backgroundTasksThreadPool,
-              storageOptions.getWriteChannelOptions(),
-              resourceId,
-              options,
-              writeConditions) {
-            @Override
-            public Storage.Objects.Insert createRequest(InputStreamContent inputStream)
-                throws IOException {
-              return initializeRequest(
-                  super.createRequest(inputStream), resourceId.getBucketName());
-            }
-          };
-    }
+
+    channel =
```

Comment:
Please make assignment part of the declaration

Reply:
updated